pom.xml

```diff
@@ -18,6 +18,11 @@
 	</repositories>
 
 	<dependencies>
+		<dependency>
+			<groupId>com.fasterxml.jackson.core</groupId>
+			<artifactId>jackson-databind</artifactId>
+			<version>2.9.9</version>
+		</dependency>
```

Comment:
Just to make sure - is this completely backwards compatible? Like changing the version needs to changes to our code?

Reply:
I've tested it and it works.

Reply:
Thanks :)